src/WordPress/Blueprints/Resources/Resolver/ResourceResolverInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace WordPress\Blueprints\Resources\Resolver;
+
+use WordPress\Blueprints\Model\DataClass\ResourceDefinitionInterface;
+
+interface ResourceResolverInterface
+{
+    public function parseUrl(string $url): ?ResourceDefinitionInterface;
+
+    public function supports(ResourceDefinitionInterface $resourceDefinition): bool;
```

Comment:
This was also renamed from `$resource` to `$resourceDefinition`. That's a lot of changes for one PR. Let's stick to PHP 7.4 downgrade and move all the renames to another PR so this one can get merged faster.

Reply:
Reverted.